.dockerignore

```diff
@@ -0,0 +1,9 @@
+# Ignore everything...
+*
+# ... except:
+!bin/
+!lib/
+!chp-docker-entrypoint
+!index.js
+!package*.json
+!LICENSE
```

Comment:
I added this file because I noted that we copied things from node_packages and felt that it could risk causing some issue, and that it would trigger rebuilds for no use potentially.

I got a warning about the node_packages being copied and existing, this is what led me to this change. It will come with the drawback of needing to update this as well if we add something to include, but hopefully we will add that within the lib folder.

Reply:
I think instead of setting what gets added in dockerignore, we should whitelist in COPY directives instead. What do you think?

Reply:
Ah I did that first, and then switched to this option for aesthetics.

When I concluded it was required to look something like below, with three lines instead of one, I felt better about a `.dockerignore` and `COPY . /dest` than multiple `COPY ` statements - one for each folder and one of the files. It turned out that one cannot copy an entire folder next to files files in a COPY statement, then the contents of the folder is copied to the destination rather than the entire folder.

```Dockerfile
# the option to a .dockerignore file afaik
COPY our-bin-folder /destination
COPY our-lib-folder /destination
COPY list-of-individual-files /destination
```

Let me know what you think, then I'll go with that!

Reply:
Ping @minrk

Reply:
I added a comment about the .dockerignore file the risk of missing out on what goes on is minimized.